crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1662,17 +1662,13 @@ impl Client {
         self.send(request, None).await
     }
 
-    /// Create a room using the `RoomBuilder` and send the request.
+    /// Create a room with the given parameters.s
     ///
-    /// Sends a request to `/_matrix/client/r0/createRoom`, returns a
-    /// `create_room::Response`, this is an empty response.
-    ///
-    /// # Arguments
-    ///
-    /// * `room` - The easiest way to create this request is using the
-    /// `create_room::Request` itself.
+    /// Sends a request to `/_matrix/client/r0/createRoom`, returns the created
+    /// room as a [`room::Joined`] object.
```

Comment:
Perhaps we should link to the `create_dm()` method from here. I know they are one above the other, but that isn't guaranteed to stay this way and people might not have enough screen real estate to see both of them.